### ISSUE TITLE
bodymovin Stuck on exporting images for over 30mins

### ISSUE THREAD

**Issue by HamedGhaithan** [airbnb/lottie-web#2010]
Hi, sorry for removing the instruction messege, 
so basically I was making an animation of a logo that I've imported from photoshop. 

Whenever I press the render button it starts normaly but then it gets stuck on the exporting iamage messege. 
I've enabled the scripts to write files. 
My render queue is empty aswell. 

Please I need help! 
![salla](https://user-images.githubusercontent.com/61466821/75260924-52cb5600-5810-11ea-97ba-53522bb8e692.PNG)

**Comment by bodymovin** [comment]
Hi, what version of After Effects are you using?

**Comment by HamedGhaithan** [comment]
Hello, I'm using version 15.0

**Comment by bodymovin** [comment]
Yes, unfortunately there seems to be a problem exporting images with AE older than 16

**Comment by HamedGhaithan** [comment]
Yeah, I went ahead and updated AE and the problem was fixed. 

Thank you.

**Comment by thomas4607** [comment]
Got the same issue, tried the same thing. Render Queue is empty, scripts enabled. AE version 17.5.1 (current build)

**Comment by nihey** [comment]
I've experienced this issue recently in After Effects 2022 (22.1.1 - Build 74), bodymovin [v5.8.1](https://github.com/airbnb/lottie-web/releases/tag/v5.8.1), disabling image compression seems to have fixed this issue.

**Comment by ntgladia** [comment]
> I've experienced this issue recently in After Effects 2022 (22.1.1 - Build 74), bodymovin [v5.8.1](https://github.com/airbnb/lottie-web/releases/tag/v5.8.1), disabling image compression seems to have fixed the issue.
nihei +1 i struggle for 3 days finding answer upgrades everything does not work. but when unchecked compression it suddenly works. the only thing is it does not export the image sequence.

**Comment by U70U** [comment]
**i have this problem to in 2024**
![image](https://github.com/user-attachments/assets/4e1dd01d-16c6-469e-8dab-fc04a7149c53)
how can this problem fixed ?
